### PR TITLE
added pagination and api annotations

### PR DESCRIPTION
**Pull request by BlueVioletTeti**

### CODE REVIEW

**Review by holsoni — APPROVED** [review]
LGTM!

File: pom.xml [hunk 1]
```diff
@@ -76,8 +76,16 @@
 			<artifactId>hibernate-validator</artifactId>
 			<version>8.0.1.Final</version>
 		</dependency>
-
-
+		<dependency>
+			<groupId>org.springframework.boot</groupId>
+			<artifactId>spring-boot-starter-validation</artifactId>
+			<version>3.1.3</version>
```
Comment (nacenik) [inline pom.xml]: do you really need a version here?

File: pom.xml [hunk 2]
```diff
@@ -76,8 +76,16 @@
...
+		</dependency>
+		<dependency>
+			<groupId>org.springdoc</groupId>
+			<artifactId>springdoc-openapi-starter-webmvc-ui</artifactId>
+			<version>2.2.0</version>
```
Comment (nacenik) [inline pom.xml]: put this as variable inside properties

**Review by nacenik — CHANGES_REQUESTED** [review]
Good solution, let's improve it a little bit :)

**Review by nacenik — APPROVED** [review]
LGTM! 👍

**Pull request merged**